pkg/framework/test/examples_api_server_custom_pathfinder_test.go

```diff
@@ -0,0 +1,19 @@
+package test_test
+
+import (
+	"fmt"
+
+	. "k8s.io/kubectl/pkg/framework/test"
+)
+
+func ExampleAPIServer_specialPathFinder() {
+	apiServer := &APIServer{
+		PathFinder: myCustomPathFinder,
```

Comment:
I still don't understand why this is more complicated than it has to be?

My understanding is that you want lazy-finding of the binary. Do you assume that the decision can't be made at APIServer *creation* time, but can be made at APIServer *start* time? Why would we expect that to be different? If we don't, we can just let the user compose however they want. That would also create a more gradual learning curve (*I don't need to know new types if I just want to pass in an absolute path*, *I can understand how the APIServer works by looking just at its interface*)

Reply:
Actually, this example wouldn't be needed at all then.

Reply:
Agreed -- addressed in #184